stable/mongodb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mongodb
-version: 5.14.2
+version: 5.15.2
```

Comment:
When bumping the minor version, please reset the patch version to "0". The correct versioning should be `5.15.0`

Reply:
@juan131 I was confused by this initially, thanks for clarifying this. Fixed with new commit :ballot_box_with_check: